src/schema/nodes/keywords_section.ts

```diff
@@ -36,14 +36,14 @@ export const keywordsSection: NodeSpec = {
   selectable: false,
   parseDOM: [
     {
-      tag: 'section.keywords',
+      tag: 'div.keywords',
```

Comment:
what is the purpose of replacing 'section' with 'div'?

Reply:
This is to differentiate the keywords section from normal sections and to apply different styling for section title (no numbers, smaller font size,...).

Reply:
I'm a bit concerned about recycling old keywords stuff for the new ones, hopefully that won't lead to unexpected bugs.